Cargo.toml

```diff
@@ -7,8 +7,8 @@ authors = ["Polygon Avail Team"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-kate-recovery = { git = "https://github.com/maticnetwork/avail-core.git", tag = "v0.7.0" }
-avail-subxt = { git = "https://github.com/maticnetwork/avail.git", tag = "v1.3.0" }
+kate-recovery = { git = "https://github.com/maticnetwork/avail-core.git", tag = "da-primitives/v0.4.2" }
```

Comment:
Is picking `da-primitives` tag mistake or by intention?

Reply:
Fixed :+1: